WcaOnRails/db/migrate/20160502113530_add_preferred_events_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddPreferredEventsToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :preferred_event_ids, :text
```

Comment:
> I forgot that Jeremy suggested to use serialized Array, what do you think about it? 

It sucks. `eventSpecs` already causes trouble: https://github.com/cubing/worldcubeassociation.org/issues/95

> Are there still benefits from using the separate table which I'm missing?

Yes, the data can be queried easier. E.g. How would you find all people who prefer 3x3x3 as an event? Grabbing all users and doing `includes?` in Ruby? String magic within MySQL (simple substring expressions don't work since there's `333` and `333bf`)? `PreferredEvents.where(event_id: '333')` sounds way more reasonable.

You can also avoid duplicates by using unique indices without doing some extra work in Ruby.